packages/vuetify/src/composables/layout.ts

```diff
@@ -271,7 +271,7 @@ export function createLayout (props: { overlaps?: string[], fullHeight?: boolean
         const styles = {
           [position.value]: 0,
           zIndex: zIndex.value,
-          transform: `translate${isHorizontal ? 'X' : 'Y'}(${(active.value ? 0 : -110) * (isOppositeHorizontal || isOppositeVertical ? -1 : 1)}%)`,
```

Comment:
110% was so the shadow doesn't still show up when something is hidden

Reply:
Added a check for setting isFlat based on fullyHide and isActive.